emails/donations_paused.spt

```diff
@@ -0,0 +1,16 @@
+% if donations_paused
+{{ _("Liberapay donation paused") }}
+% else
+{{ _("Liberapay donation resumed") }}
+% endif
+
+[---] text/html
+% if donations_paused
+<p>{{ _("Due to changes recently made by {0}, your donations for {0} have been temporarily paused.", recipient) }}</p>
+% else
+<p>{{ _("Due to changes recently made by {0}, your donations for {0} are set to resume.", recipient) }}</p>
+% endif
+
+<p>{{ _("View your updated donations page at the following link:") }}</p>
+
+<p><a href="{{ donations_url }}" style="{{ button_style('primary') }}">{{ _("View Donations Page") }}</a></p>
```

Comment:
I'm not sure that including a link to the Giving page in this notification is a good idea, but if there is one then it should be more like this:

```suggestion
<p><a href="{{ participant.url('giving/') }}" style="{{ button_style('default') }}">{{ _("Manage your donations") }}</a></p>
```

(Passing the URL of the Giving page as a context variable is inconsistent and potentially problematic.)